fcrepo-kernel-impl/src/test/java/org/fcrepo/kernel/impl/models/ResourceFactoryImplTest.java

```diff
@@ -208,7 +208,7 @@ public void getResource_BasicContainer_WithParent() throws Exception {
 
         when(psSession.getHeaders(parentId, null)).thenReturn(parentHeaders);
 
-        final var resc = factory.getResource(mockTx, fedoraId);
+        final var resc = factory.getResource(mockTx.getId(), fedoraId);
```

Comment:
Seems like all the `factory.getResource(mockTx.getId(), ...` in this test could be reverted back to `factory.getResource(mockTx, ...`

Reply:
I don't think these ones got reverted

Reply:
They should be good now.